android/build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
 
     dbFlowVersion = '4.2.4'
     buildToolsVersion = '33.0.0'
-    kotlinVersion = '1.6.10'
+    kotlinVersion = '1.8.10'
```

Comment:
Let's bump up the kotlinVersion in all the modules, this way the stripe module will play nice with our core in the bare workflow.